Skills.md

```diff
@@ -4,22 +4,22 @@ https://api.careerbuilder.com/core/tagging/skills
 
 This API supports the HTTP/GET and HTTP/POST methods.  
 
-This API currently supports versions 4.1 and 4.2. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.2 contains the latest taxonomy and incorporates feedback received about specific terms since the release of V4/4.1.
+This API currently supports versions 4.1, 4.2, and 5.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 5.0 contains the latest taxonomy and incorporates feedback received about specific terms since the release of V4/4.1 and V4.2.

```

Comment:
I wound like to change the example to `Accept:application/json;version=5.0`
shouldn't it be "the feedback"?

Reply:
I wound like to elaborate the "feedback" a bit more. A major difference v5 has is that it returns the only exact (or partially exact match) if that exists. In such cases, user may expect less but more accurate results. I think exposure of such information in doc is useful to customers.

Reply:
I think we can talk about the specific improvements in v5 in the release note. I don't think we go into detail about improvements in successive versions anywhere else. I'd rather not make this a changelog of R&D's improvements, in general it's good enough in my opinion just to say its getting better with each version.

Reply:
the "ex" should just be like, "version={VERSION_NUMBER}" so we don't have to keep updating it :) and I would do it in code-font too (with the backticks) so it's clearer that we're literally showing what your code should look like.

"incorporates feedback" is grammatically correct, no "the" needed.

+1 on not making it a changelog of R&D's improvements. if they want to start publishing such a document, we can certainly link to it here.

Reply:
Historical beef comment: Is 'Ex.' a common abbreviation for example? Could we change it to 'e.g.'?